vanilla-framework/vanilla-brochure-theme
# Update strip padding on small screens

## Done
Updated the small screen padding for `is-deep` and `is-shallow`. Added padding for the default `p-strip`.

## QA
- Pull down branch and run `gulp jekyll`
- Go to each strip and make sure the padding matches [the spec](https://github.com/ubuntudesign/vanilla-brochure-theme-design/blob/master/Strip/strip.md).

## Details
Fixes https://github.com/vanilla-framework/vanilla-brochure-theme/issues/80
